src/main/java/co/cask/http/NettyHttpService.java

```diff
@@ -285,9 +285,14 @@ protected void shutDown() throws Exception {
         ExecutorUtil.terminate(executionHandler.getExecutor());
       }
     }
-    LOG.info("Stopped {} http service on address {}", serviceName, bindAddress);
+    LOG.debug("Stopped HTTP Service {} on address {}", serviceName, bindAddress);
   }
 
+  public String getSimpleServiceName(String serviceName) {
+    return serviceName.substring(serviceName.lastIndexOf(".") + 1);
```

Comment:
@chtyim debug logs can carry the qualified service name this way, in case its ever required.